sdk/storage/Azure.Storage.Files.DataLake/README.md

```diff
@@ -191,23 +191,27 @@ subDirectoryClient.Create();
 DataLakeFileClient fileClient = subDirectoryClient.GetFileClient(Randomize("sample-file"));
 fileClient.Create();
 
+// Create progress handler to observe operation and collect paths that failed to change Access Controls.
+// Customer is expected to provide own implementation.
+IProgress<Response<AccessControlRecursiveChanges>> progress = null;
```

Comment:
@tg-msft That be ideal. However, I was struggling to create concise implementation doing that. Unfortunately, IProgress didn't let me provide lambda/anonymous implementation (which would be quite simple and elegant in Java btw).
That makes we wonder if we should use Action instead of IProgress.
What do you think?

Reply:
I think we want `IProgress`.  Just use https://docs.microsoft.com/en-us/dotnet/api/system.progress-1.-ctor?view=netframework-4.8#System_Progress_1__ctor_System_Action__0__ for the example.

Reply:
Ops. That's an oversight on my side. Thank you @tg-msft . I wonder what I was typing into search that it didn't come in the result. That is going to simplify quite a lot of things.